test/module/irohad/consensus/yac/yac_fixture.hpp

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_YAC_MOCKS_HPP
+#define IROHA_YAC_MOCKS_HPP
+
+#include <gtest/gtest.h>
+
+#include "consensus/yac/cluster_order.hpp"
+#include "consensus/yac/yac.hpp"
+
+#include "module/irohad/consensus/yac/mock_yac_crypto_provider.hpp"
+#include "module/irohad/consensus/yac/mock_yac_network.hpp"
+#include "module/irohad/consensus/yac/mock_yac_timer.hpp"
+#include "module/irohad/consensus/yac/yac_test_util.hpp"
+
+namespace iroha {
+  namespace consensus {
+    namespace yac {
+
+      class YacTest : public ::testing::Test {
+       public:
+        // ------|Network|------
+        std::shared_ptr<MockYacNetwork> network;
+        std::shared_ptr<MockYacCryptoProvider> crypto;
+        std::shared_ptr<MockTimer> timer;
+        std::shared_ptr<Yac> yac;
+
+        // ------|Round|------
```

Comment:
Not very clear, why collection of peers is under the comment "Round". Maybe expand or change this title?